### ISSUE TITLE
Cannot download an entire folder

### ISSUE THREAD

**Issue by alr2413** [loudKode/Uptobox-Client#3]
About this issue:
If you right click on a folder and select the download button, the following error occurs:

```
System.Exception: Error converting value "bad file_code" to type 'UptoboxSDK.JSON.JSON_GenerateDownloadLinkData'. Path 'data', line 1, position 68.
   at UptoboxSDK.UClient.VB$StateMachine_35_GET_GenerateDownloadLink.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Uptobox_Client.MainForm.VB$StateMachine_30_MB_Download_Click.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)

```

**Comment by loudKode** [comment]
download folder not allowed 
maybe in next release i add this feature

**Comment by alr2413** [comment]
Ok. I think for this feature you only need to iterate over the files in folder & download each of them.
Thanks.

**Comment by loudKode** [comment]
added

**Comment by alr2413** [comment]
Thanks. 
But still there is a problem. If total size of items in the selected folder was big, the following exception occurred:

![Screen Shot 2019-11-08 at 4 38 15 PM](https://user-images.githubusercontent.com/10116275/68479161-f1727d80-0246-11ea-856a-410570ba3d12.png)

**Comment by loudKode** [comment]
fixed in version 1.7.2
